### PR TITLE
Removes a terrible balance decision from the speed potions

### PR DESCRIPTION
**Pull request by redmoogle**
# General Documentation

### Intent of your Pull Request

Red slime core or however the fuck you make speed potion nolonger blow you up or spawn gondola pods

### Why is this change good for the game?

Shit balance, just nerf the speed effect which it did, so it's only logical to remove this nerf

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
Slime extract works consistently

### What general grouping does this PR fall under? 
Xenobio buff

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 
100% spawnrate for potion

# Changelog

:cl:  
tweak: Red slime cores nolonger explode or spawn gondola pods
/:cl:

### CODE REVIEW

**Comment by patpol4** [comment]
Why was this a thing haha

**Review by SomeguyManperson — APPROVED** [review]

**Comment by ToasterBiome** [comment]
That shit was funny though
